lms/envs/common.py

```diff
@@ -2118,14 +2118,16 @@
 
 # JWT Settings
 JWT_AUTH = {
-    'JWT_SECRET_KEY': None,
+    # TODO Set JWT_SECRET_KEY to a secure value. By default, SECRET_KEY will be used.
+    # 'JWT_SECRET_KEY': '',
```

Comment:
FYI @edx/devops  They are now! 

Seriously, this should not affect site maintainers actively using JWT because we already override the values. Those who are not actively using JWT will remain unaffected (for now). My OAuth changes in #12194 will effectively force everyone to use JWT at which point this becomes everyone's problem...I should probably put a feature flag around that!

@bderusha is working on moving us to use asymmetric signing keys to avoid the coupling you mention. We definitely do not want the edxapp secret key shared with the IDAs and other clients.

FYI @lamagnifica this will definitely need to go into the release notes!